src/TodoListPage.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useCallback, useState } from 'react';
+
+import TodoInput from './TodoInput';
+import TodoList from './TodoList';
+
+function TodoListPage() {
+  const [TodoState, setTodoState] = useState({
+    todo: '',
+    todoList: [],
+  });
+
+  const { todo, todoList } = TodoState;
+
+  const handleTodoInput = useCallback(
+    (e) => {
+      setTodoState({ ...TodoState, todo: e.target.value });
+    },
+    [TodoState],
+  );
+
+  const handleAddTodo = useCallback(() => {
+    setTodoState({ todo: '', todoList: todoList.concat(todo) });
+  }, [TodoState]);
+
+  const handleDeleteTodo = useCallback(
+    (indexToDelete) => {
+      setTodoState({
+        todo: '',
+        todoList: todoList.filter(
+          (todoText) => todoText !== todoList[indexToDelete],
+        ),
```

Comment:
추후 확장성을 고려해서 영향을 받지 않는 값들은 이렇게 처리할 수 있을 것 같네요.

```suggestion
      setTodoState({
        ...TodoState,
        todoList: todoList.filter(
          (todoText) => todoText !== todoList[indexToDelete],
        ),
```

Reply:
완료한 Todo를 삭제할 때는 input value 를 초기화할 필요는 없었네요 피드백 주신대로 수정해보겠습니다!